javatests/arcs/android/storage/service/BindingContextTest.kt

```diff
@@ -80,9 +85,10 @@ class BindingContextTest {
   }
 
   private fun buildContext(
+    storeProvider: suspend () -> ActiveStore<*, *, *> = { store },
```

Comment:
Why not make this `store: ActiveStore<*, *, *> = store`, and then below in line 85 construct the `storeProvider`. That way you make it ever so slightly easier for callers to call this method.